src/SQLQueryStress/Form1.cs

```diff
@@ -618,5 +607,107 @@ private void ExportBenchMarkToFile(string fileName)
                     string.Format("There was an error saving the benchmark to '{0}', make sure you have write privileges to that path",fileName));
             }
         }
+
+        private void WriteBenchmarkTextContent(TextWriter tw)
+        {
+            tw.WriteLine(string.Format("Test ID: {0}",
+                                _testGuid));
+            tw.WriteLine(string.Format("Test TimeStamp: {0}",
+                                _testStartTime));

```

Comment:
use the "s" format for dates

Reply:
Just to have a record of history: 
I tried the s format and it prints the date as "2019-07-23T08:57:03". 
While it looks more standard and ISO compliant, it does not play well with excel. 
Excel recognises data in that column as text instead of datetime. 
So I left it as it is.